addons/stock/views/stock_orderpoint_views.xml

```diff
@@ -60,6 +60,8 @@
                 <field name="product_max_qty" string="Max" optional="show"/>
                 <field name="qty_multiple" optional="hide"/>
                 <field name="qty_to_order"/>
+                <field name="qty_to_order_manual" column_invisible="True"/>
+                <button name="action_remove_manual_qty_to_order" type="object" icon="fa-undo" title="Remove manually entered value and replace by the quantity to order based on the forecasted quantities" invisible="not qty_to_order_manual"/>
```

Comment:
In terms of functionality, would it be better to automatically reset the qty_to_order_manual after  it has been used once ?

Reply:
Yes great idea, I added this in :+1: